src/routes/app/+layout.svelte

```diff
@@ -41,6 +45,11 @@
       icon: ShareIcon,
     },
   ];
+
+  // Get active calendar to make sure we have received the calendar_created event
+  let activeCalendar = liveQuery(() =>
```

Comment:
I've made a change today which means the calendar _is_ actually present in the database before we receive "calendar_created" event, but it's `name` field is not set. This is due to me trying to reduce complexity around calendar/stream duplicating some of the same state. I've now merged them into one database table, where the calendar has a `stream` field. We create the calendar whenever we resolve an invite code, and then set it's name and other fields when the "calendar_created" event arrives.

The only change which will be required here is to make the conditional check below be for `activeCalendar.name`.